formidable/forms/__init__.py

```diff
@@ -50,10 +50,27 @@ def __init__(self, *args, **kwargs):
         super(BaseDynamicForm, self).__init__(*args, **kwargs)
         self._bound_fields_cache = FormidableBoundFieldCache()
 
+    def rule_has_empty_fields(self, rule, data):
```

Comment:
Why not put this kind of check in the rule itself ? Maybe for specific needed the rules can override its method in order to configure a specific check.